File: app/src/main/java/com/reteno/sample/CustomReceiverPushReceived.java
package com.reteno.sample;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.reteno.util.Logger;
import com.reteno.util.UtilKt;

public class CustomReceiverPushReceived extends BroadcastReceiver {

    private static String TAG = CustomReceiverPushReceived.class.getSimpleName();

    @Override
    public void onReceive(Context context, Intent intent) {
        Logger.i(TAG, "onReceive(): ", UtilKt.toStringVerbose(intent.getExtras()));
    }
}

File: app/src/main/java/com/reteno/sample/CustomReceiverPushReceived.kt
package com.reteno.sample

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.reteno.core.util.Logger
import com.reteno.core.util.toStringVerbose

class CustomReceiverPushReceived : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Logger.i(TAG, "onReceive(): ", intent.extras.toStringVerbose())
    }

    companion object {
        private val TAG = CustomReceiverPushReceived::class.java.getSimpleName()
    }
}
